modules/serverless-installing-cli-web-console.adoc

```diff
@@ -25,7 +25,7 @@ $ kn: No such file or directory
 
 .Procedure
 
-. Download the `kn` CLI from the *Command Line Tools* page. You can access the *Command Line Tools* page by clicking the image:../images/question-circle.png[title="Help"] icon in the top right corner of the web console and selecting *Command Line Tools* in the drop-down menu.
+. Download the Knative (`kn`) CLI from the *Command Line Tools* page. You can access the *Command Line Tools* page by clicking the image:../images/question-circle.png[title="Help"] icon in the top right corner of the web console and selecting *Command Line Tools* in the drop-down menu.
```

Comment:
Images can be difficult for translation and present a usability and accessibility issues. Also for usability and accessibility, avoid directional words like "top, bottom, right, left" 

Consider rephrasing to, `You can access the **Command Line Tools** page by clicking the question mark icon of the web console and selecting **Command Line Tools** in the drop-down menu.`